uber/custom_tags.py

```diff
@@ -10,8 +10,9 @@ def date_filter(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
     # TODO: if this is a datetime.time object, do we have to correct for timezone with .astimezone()?
     return ' '.join(dt.strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm') if dt else ''
 
+
 # TODO: fix this to work with JINJA2
```

Comment:
These TODOs were added during the merge, but I left them in because I didn't actually test if the end results were correct.